bundles/org.openhab.binding.openthermgateway/src/main/java/org/openhab/binding/openthermgateway/internal/DataItemGroup.java

```diff
@@ -0,0 +1,171 @@
+package org.openhab.binding.openthermgateway.internal;
+
+import java.util.HashMap;
+
+public class DataItemGroup {
+    private int id;
+    private DataItem[] dataItems;
+
+    public int getID() {
+        return id;
+    }
+
+    public void setID(int id) {
+        this.id = id;
+    }
+
+    public DataItem[] getDataItems() {
+        return dataItems;
+    }
+
+    public void setDataItem(DataItem[] dataItems) {
+        this.dataItems = dataItems;
+    }
+
+    public DataItemGroup(int id, DataItem... dataItems) {
+        this.id = id;
+        this.dataItems = dataItems;
+    }
+
+    public static final HashMap<Integer, DataItem[]> dataItemGroups = createDataItemGroups();
+
+    private static HashMap<Integer, DataItem[]> createDataItemGroups() {
+        HashMap<Integer, DataItem[]> g = new HashMap<Integer, DataItem[]>();
```

Comment:
Am I right that this `HashMap` is the same for all things? You should use a `static final HashMap` then.